app/src/main/java/com/h/pixeldroid/objects/Status.kt

```diff
@@ -100,7 +100,7 @@ data class Status(
 
     fun getUsername() : CharSequence {
         var name = account.display_name
-        if (name.isEmpty()) {
+        if (name.isNullOrEmpty()) {
```

Comment:
Is name really nullable? Also we should try to get rid of the var here, and take the `return` statement out of the if

Reply:
Everything from the objects packages is nullable, even if it's not declared as such (the library can put null values in there regardless)

I changed this because there was a response where it was null.